packages/module/src/DataViewToolbar/DataViewToolbar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Pagination, Toolbar, ToolbarContent, ToolbarItem } from '@patternfly/react-core';
+import { DataViewPaginationProps } from '../Hooks';
+
+export interface DataViewToolbarProps extends DataViewPaginationProps {
+  /** Toolbar className */
+  className?: string;
+  /** Flag indicating that toolbar is used below the data part */
+  isBottom?: boolean;
+  /** Custom OUIA ID */
+  ouiaId?: string;
+}
+
+export const DataViewToolbar: React.FunctionComponent<DataViewToolbarProps> = ({ className, ouiaId = 'DataViewToolbar', page, perPage, isBottom }: DataViewToolbarProps) => (
+  <Toolbar ouiaId={ouiaId} className={className}>
+    <ToolbarContent>
+      {page && perPage && (
```

Comment:
I think we might want to pass a prop `pagination` that will have the object shape. That way we can have a bit more strict typing while keeping the feature optional.